app/uk/gov/hmrc/plasticpackagingtax/registration/views/group/group_member_already_registered_page.scala.html

```diff
@@ -0,0 +1,49 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.plasticpackagingtax.registration.models.request.JourneyRequest
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.html.main_template
+@import uk.gov.hmrc.plasticpackagingtax.registration.controllers.group.{routes => groupRoutes}
+@import uk.gov.hmrc.plasticpackagingtax.registration.views.model.Title
+
+@this(
+        govukLayout: main_template,
+        pageHeading: pageHeading,
+        paragraphBody: paragraphBody,
+        paragraph: paragraph,
+        link:link
+)
+
+
+@()(implicit request: JourneyRequest[_], messages: Messages)
+
+@organisationName = @{request.registration.organisationDetails.businessName.getOrElse("Your organisation")}
```

Comment:
This is not the right organisation name - I think it needs to be the name of the member the user just tried to add.